README.md

```diff
@@ -50,7 +49,7 @@ This daemon runs as root. It provides the ability to export existing RBD images
 
 7. Start the gateway server daemon:
 		
-		$ python3 nvme_gw_server.py [-c config_filename]
+		$ python3 -m control [-s settings_file]
```

Comment:
nit: Do we really need the server (and cli) as a module? Starting application like this looks a little weird to me. It is not that I care much but anyway...

Reply:
Yes, I guess the server and cli don't really need to be modules since they have no reuse anyway (i.e. will not be imported anywhere else). Or is there any other reason you made them modules @sskaur ? I mean we can still have them in their own directories without them being modules.

Reply:
I had them as modules because I thought it might make it easier to import for finer grained unit tests in the future. There's also `make run` for starting the server, but I see your point with cli. I could pull out their respective argument parsing (in each __main__) and move them into scripts to run at the top level?

Reply:
Good point about unit tests.

`make run` is good for development but not for production.

Actually I am fine with any solution right now as it may be improved later. Eventually I suppose we will need some automatic "entry point" script creation like [1].

[1] https://setuptools.pypa.io/en/latest/userguide/quickstart.html#entry-points-and-automatic-script-creation